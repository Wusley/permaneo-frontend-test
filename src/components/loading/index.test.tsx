import { render } from '@testing-library/react'
import { Loading } from './'

describe( 'Loading Component', () => {
  it( 'matches the snapshot with default props', () => {
    const { container } = render( <Loading /> )
    expect( container ).toMatchSnapshot()
  } )

  it( 'matches the snapshot with custom props', () => {
    const { container } = render( <Loading color="blue.500" height="200px" marginY={4} size="80px" /> )
    expect( container ).toMatchSnapshot()
  } )
} )