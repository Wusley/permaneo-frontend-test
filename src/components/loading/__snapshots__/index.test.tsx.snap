// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component matches the snapshot with custom props 1`] = `
<div>
  <div
    class="css-rc9w4c"
  >
    <div
      aria-valuemax="100"
      aria-valuemin="0"
      class="chakra-progress css-u1nf2k"
      data-indeterminate=""
      role="progressbar"
    >
      <svg
        class="css-xmlx29"
        viewBox="0 0 100 100"
      >
        <circle
          class="chakra-progress__track css-1bhxzgy"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
        <circle
          class="chakra-progress__indicator css-p6i8i8"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
      </svg>
    </div>
  </div>
</div>
`;

exports[`Loading Component matches the snapshot with default props 1`] = `
<div>
  <div
    class="css-17orizo"
  >
    <div
      aria-valuemax="100"
      aria-valuemin="0"
      class="chakra-progress css-120wkjd"
      data-indeterminate=""
      role="progressbar"
    >
      <svg
        class="css-jf70f3"
        viewBox="0 0 100 100"
      >
        <circle
          class="chakra-progress__track css-1bhxzgy"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
        <circle
          class="chakra-progress__indicator css-1b60oz0"
          cx="50"
          cy="50"
          r="42"
          stroke-width="10px"
        />
      </svg>
    </div>
  </div>
</div>
`;
